descope/management/access_key.py

```diff
@@ -120,21 +123,22 @@ def update(
         self,
         id: str,
         name: str,
+        description: Optional[str] = None,
     ):
         """
-        Update an existing access key with the given various fields. IMPORTANT: All parameters are used as overrides
-        to the existing access key. Empty fields will override populated fields. Use carefully.
+        Update an existing access key with the given various fields. IMPORTANT: id and name are mandatory fields.
```

Comment:
i'd add in the comment that if description is not provided, it will not be overriden